packages/features/bookings/lib/handleConfirmation.ts

```diff
@@ -83,7 +83,27 @@ export async function handleConfirmation(args: {
     paid,
     emailsEnabled = true,
     platformClientParams,
+    traceContext,
   } = args;
+
+  const confirmationMeta = {
+    bookingId,
+    eventTypeId: booking.eventTypeId,
+    eventTypeTitle: booking.eventType?.title,
+    userId: booking.userId,
+    recurringEventId,
+    paid,
+    emailsEnabled,
+    platformClientId: platformClientParams?.platformClientId,
+  };
+
+  const spanContext = traceContext
+    ? distributedTracing.createSpan(traceContext, "handle_confirmation", confirmationMeta)
+    : distributedTracing.createTrace("handle_confirmation_fallback", {
+        meta: confirmationMeta,
+      });
```

Comment:
Same here I would n't want this logic here, traceContext could always be considered available.